### PR TITLE
node upgrade changes.

### PR DESCRIPTION
**Pull request by supritat**

### CODE REVIEW

File: installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh [hunk 1]
```diff
@@ -3,7 +3,6 @@
 securityGroupIds=$1
 subnetIds=$2
 iamRoleARN=$3
-region=$4
```
Comment (rajeevr2715) [inline installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh]: Region should be there.

Comment (supritat) [reply installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh]: Included region

Comment (svidhani) [reply installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh]: why, its not being used. shellcheck throws an error

Comment (rajeevr2715) [reply installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh]: correct. you can remove it.

Comment (svidhani) [reply installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh]: region=$4
^-- SC2034: region appears unused. Verify it or export it.

**Pull request merged**